src/lib/registry/notifier-registry.js

```diff
@@ -0,0 +1,46 @@
+const Notifier = require('../notifier');
+const { throwIf } = require('../utils/type-checks');
+
+const EmptyRegistry = require('./empty-registry');
+
+class NotifiersRegistry extends EmptyRegistry {
+    constructor() {
+        super();
+    }
+
+    add(name, instance) {
+        throwIf.not.string(name, 'name');
+        throwIf.not.constructor(instance, 'instance');
+        this._registry[name] = instance;
+    }
+
+    register(name, ctor) {
+        throwIf.not.string(name, 'name');
+        throwIf.not.constructor(ctor, 'ctor');
+
+        const nameUpper = name.toUpperCase();
+        const nameLower = name.toLowerCase();
+
+        this._registry[nameLower] = this._registry[nameUpper] = ctor;
+
+        return this;
+    }
+
+    get(query, opts = {}) {
+        if (query instanceof Notifier) return query;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
        if (query instanceof Notifier) {return query;}
```